.atomist/manifest.yml

```diff
@@ -1,6 +1,6 @@
 group: atomist-rugs
 artifact: common-editors
-version: "0.7.0"
-requires: "[0.8.0,1.0.0)"
+version: "0.8.0"
```

Comment:
so we leave it as the old version even though the old version has been published already??